site/it/README.md

```diff
@@ -0,0 +1,36 @@
+# Traduzioni della comunità
+
+La nostra comunità di Tensorflow ha tradotto questi documenti. Poichè questa traduzioni della comunità sono *best-effort*, non c'è garanzia che questa sia un riflesso preciso e aggiornato della [documentazione ufficiale in inglese](https://www.tensorflow.org/?hl=en). 
+Se avete suggerimenti per migliorare questa traduzione, mandate per favore una pull request al repository Github [tensorflow/docs](https://github.com/tensorflow/docs). 
+Per proporsi come volontari alla scrittura o alla review delle traduzioni della comunità contattate la 
+[mailing list docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
+
+Nota: Per favore concentrate i lavori di traduzione su 
+[TensorFlow 2](https://www.tensorflow.org) nella cartella
+[site/en/](https://github.com/tensorflow/docs/tree/master/site/en/). I documenti della comunita per TF 1.x non saranno più aggiornati mentre prepariamo la release 2.0. Vedere l'annuncio 
+[seguente](https://groups.google.com/a/tensorflow.org/d/msg/docs/vO0gQnEXcSM/YK_ybv7tBQAJ).
+
+# Community translations
+
+Our TensorFlow community has translated these documents. Because community
+translations are *best-effort*, there is no guarantee that this is an accurate
+and up-to-date reflection of the
+[official English documentation](https://www.tensorflow.org/?hl=en). 
+If you have suggestions to improve this translation, please send a pull request 
+to the [tensorflow/docs](https://github.com/tensorflow/docs) GitHub repository. 
+To volunteer to write or review community translations, contact the
+[docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
+
+Note: Please focus translation efforts on
+[TensorFlow 2](https://www.tensorflow.org) in the
+[site/en/](https://github.com/tensorflow/docs/tree/master/site/en/)
+directory. TF 1.x community docs will no longer be updated as we prepare for the
+2.0 release. See
+[the announcement](https://groups.google.com/a/tensorflow.org/d/msg/docs/vO0gQnEXcSM/YK_ybv7tBQAJ).
+
+# Guida alla traduzione italiana
+
+Alcune parole fanno parte di un vocabolario tecnico inglese e pertanto non hanno una traduzione naturale. Per favore *non* traducete le seguenti parole:
+
+*   layer
+*   loss
```

Comment:
Loss Is translated in the document.
Obiettivo Is good enought but loss Is also  very common

Reply:
I see, for now I am removing loss from the list of words to not translate. In the future this can change of course